jobs/quicksuggest-2-bq/quicksuggest_2_bq/main.py

```diff
@@ -0,0 +1,207 @@
+    requests_session = requests.Session()
+
+    suggestions = {}
+
+    for record in data_records:
+        attachment_url = f"{attachments_base_url}{record['attachment']['location']}"
+
+        response = requests_session.get(attachment_url)
+
+        if response.status_code != 200:
+            # Ignore unsuccessful requests for now
+            logging.error(
+                (
+                    "Failed to download attachment for record with ID '%s'."
+                    " Response status code %s."
+                ),
+                record["id"],
+                response.status_code,
+            )
+            continue
+
+        # Each attachment is a list of suggestion objects and each suggestion
+        # object contains a list of keywords. Load the suggestions into pydantic
+        # model instances to discard all fields which we don't care about here.
+        suggestions.update(
+            {
+                suggestion_data["id"]: KintoSuggestion(**suggestion_data)
+                for suggestion_data in response.json()
+            }
+        )
+
+    return suggestions
+
+
+def store_suggestions(
+    date: datetime.datetime,
+    destination_project: str,
+    destination_table_id: str,
+    kinto_suggestions: Dict[int, KintoSuggestion]
+):
+    """Get records, download attachments and return the suggestions."""
+
+    # Turn the suggestions into dicts and augment them with
+    # an insertion date.
+    suggestions = [
+        {**asdict(suggestion), "insert_date": date.date().isoformat()}
+        for suggestion in kinto_suggestions.values()
+    ]
+
+    client = bigquery.Client(project=destination_project)
+
+    job_config = bigquery.LoadJobConfig(
+        create_disposition="CREATE_IF_NEEDED",
+        schema=[
+            bigquery.SchemaField("insert_date", "DATE"),
+            bigquery.SchemaField("advertiser", "STRING"),
+            bigquery.SchemaField("click_url", "STRING"),
+            bigquery.SchemaField("iab_category", "STRING"),
+            bigquery.SchemaField("icon", "STRING"),
+            bigquery.SchemaField("impression_url", "STRING"),
+            bigquery.SchemaField("id", "INTEGER"),
+            bigquery.SchemaField("keywords", "STRING", "REPEATED"),
+            bigquery.SchemaField("title", "STRING"),
+            bigquery.SchemaField("url", "STRING"),
+        ],
+        source_format=bigquery.SourceFormat.NEWLINE_DELIMITED_JSON,
+        write_disposition="WRITE_APPEND",
+    )
+
+    load_job = client.load_table_from_json(
+        suggestions,
+        destination_table_id,
+        location="US",
+        job_config=job_config,
+    )
+
+    try:
+        load_job.result()
+    except BadRequest:
+        for e in load_job.errors:
+            logging.error(f"ERROR: {e['message']}")
+
+    stored_table = client.get_table(destination_table_id)
+    logging.info(f"Loaded {stored_table.num_rows} rows.")
+
+
+@click.command()
+@click.option(
+    "--date",
```

Comment:
It looks like we don't actually use this. If we're going to include a date field in the output, we should make sure it reflects this.

Reply:
This is currently being used in `store_suggestions` to augment the parsed suggestions with the date. But we can remove this if we no longer want it as you suggested above!